.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
   "extends": "airbnb/base",
+  "parser": "babel-eslint"
```

Comment:
It doesn't make sense for it to be in it's own PR as it literally does nothing other than add ES7 syntax support to eslint.  We don't use ES7 until this commit so the history wouldn't make much sense?